### PR TITLE
Update schema and API to make mechanism part of the unique key

### PR DESCRIPTION
**Pull request by dglemos**
Ticket: https://www.ebi.ac.uk/panda/jira/browse/G2P-162

This PR can only be merged after the following code is merged: https://github.com/EBI-G2P/gene2phenotype_api/pull/41

DB to test: dlemos_G2P_2024_v17_new_keys

### CODE REVIEW

File: gene2phenotype_project/gene2phenotype_app/views/search.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def get_queryset(self):
                     base_phenotype & Q(lgdpanel__panel__name=search_panel) |
                     base_phenotype_2 & Q(lgdpanel__panel__name=search_panel) |
                     base_g2p_id & Q(lgdpanel__panel__name=search_panel)
-                ).order_by('locus__name', 'stable_id__stable_id').distinct()
+                ).order_by('locus__name', 'disease__name').distinct()
```
Comment (olaaustine) [inline gene2phenotype_project/gene2phenotype_app/views/search.py:66]: Question - why are we ordering by disease name instead of stable id?

**Review by olaaustine — COMMENTED** [review]
Just a comment and a question, also please can you pull the new changes. 
Thanks @dglemos

File: gene2phenotype_project/gene2phenotype_app/views/search.py (hunk 1 above)
Comment (dglemos) [reply gene2phenotype_project/gene2phenotype_app/views/search.py:66]: This way the diseases with the same mechanism are displayed together. It looks more organised this way.

**Comment by dglemos** [comment]
@olaaustine thanks for the review.
This PR is up-to-date with main.

**Review by olaaustine — APPROVED** [review]
Thanks @dglemos. 
LGTM!

**Pull request merged**